src/main/java/org/datadog/jmxfetch/JMXAttribute.java

```diff
@@ -51,7 +51,8 @@
     private Boolean cassandraAliasing;
 
     JMXAttribute(MBeanAttributeInfo attribute, ObjectName beanName, String instanceName,
-            Connection connection, HashMap<String, String> instanceTags, Boolean cassandraAliasing) {
+            Connection connection, HashMap<String, String> instanceTags, Boolean cassandraAliasing,
```

Comment:
How do you feel we should handle the case where the instanceTags `host` tag is set (for whatever reason), but `emptyDefaultHostname` is true. IMHO it might make sense to just evict it from the HashMap and make the boolean option take precedence.

Reply:
For python/golang checks, we want to support an empty default hostname + check-provided hostname override (for kubernetes events node events / kube_state node metrics), so it's a legit use case.

It's currently not possible as an instance tag as [it'll be sanitized to bean_host](https://github.com/DataDog/jmxfetch/blob/master/src/main/java/org/datadog/jmxfetch/JMXAttribute.java#L165), but could it be added as a bean tag? 

If so, do you think there's a better method to plug this logic into?

Reply:
Let's leave as-is, I think its probably fine.